docs/update-cli.md

```diff
@@ -0,0 +1,19 @@
+# Update OpenAPI CLI
+
+If you've already installed OpenAPI CLI, you'll want to keep it as shiny and up to date as possible. Staying on top of versions also ensures that you get the latest functionality and bug fixes, as well as the most recent Redocly configuration file. A must-have!
+
+:::success Tip
+Ensure you have the latest version of npm before you begin.
+:::
+
+Navigate to your local `openapi-starter` folder and open `package.json` in any text editor.
+
+Find the line that contains `"@redocly/openapi-cli":` followed by the version number. Change the version number to the latest one in the Changelog, save the changes and close `package.json`.
+
+To install the latest OpenAPI CLI version, run this command in the root of your local `openapi-starter` folder:
+
+```shell Command
+npm i -g @redocly/openapi-cli@latest
```

Comment:
Right. I think we're mixing two different ways to update here, which might be my fault because I didn't explain this clearly enough to @JodyWinter. I usually update OpenAPI CLI just by running this `@latest` command from anywhere, but that's because I have it installed globally. 

So I suggest we split the text into these two ways to update: 

1. Global (for people who installed it globally, and can just run the command from anywhere); 
2. openapi-starter (for people who installed it from the openapi-starter template and need to modify the package.json file).

Would that make sense?